internal/bam/bam.go

```diff
@@ -23,16 +23,14 @@ import (
 
 	"github.com/googlegenomics/htsget/internal/bgzf"
 	"github.com/googlegenomics/htsget/internal/binary"
+	"github.com/googlegenomics/htsget/internal/csi"
 	"github.com/googlegenomics/htsget/internal/genomics"
 )
 
 const (
 	baiMagic = "BAI\x01"
 	bamMagic = "BAM\x01"
```

Comment:
@anamanolache By the way you will need the followin magic number for CSI files as well:

```Bash
$ od -c -N 4 sample-example.bcf.gz.csi
0000000   C   S   I 001
```